src/dsymbol/modulecache.d

```diff
@@ -390,13 +398,17 @@ private:
 		return r.front.modificationTime != modification;
 	}
 
-	void scanAll()
+	/**
+	 * Params:
+	 *     force = force scanning the imports
+	 */
+	void scanAll(bool force = false)
 	{
 		foreach (ref importPath; importPaths)
 		{
-			if (importPath.scanned)
+			if (!force && importPath.scanned)
 				continue;
-			scope(success) importPath.scanned = true;
+			scope(success) if(!force) importPath.scanned = true;
```

Comment:
that's because i didn't want to change it's behavior, i only needed to check wether or not things need to be recached

i'll come up with a cleaner solution